soundata/jams_utils.py

```diff
@@ -76,19 +78,57 @@ def jams_converter(
 
     # soundata tags
     if tags is not None:
-        if not isinstance(tags, annotations.Tags):
-            raise TypeError("tags should be of type annotations.Tags")
-        jam.annotations.append(tags_to_jams(tags, duration=jam.file_metadata.duration))
+        if isinstance(tags, annotations.Tags):
+            jam.annotations.append(
+                tags_to_jams(tags, duration=jam.file_metadata.duration)
+            )
+        elif isinstance(tags, annotations.MultiAnnotator):
+            jam.annotations.extend(
```

Comment:
just out of curiosity, is `extend` the same as `append`? I couldn't find it in the API

Reply:
`AnnotationArray` is a subclass of Python native `list` so maybe that's why `extend` (`list` method) is not in the JAMS API.

`[0].append([1 ,2, 3])` gives `[0, [1, 2, 3]]` but `[0].extend([1, 2, 3])` gives `[0, 1, 2, 3]`.